mne/io/curry/tests/test_curry.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: UTF-8 -*-
+#
+# Authors: Dirk Gütlin <dirk.guetlin@stud.sbg.ac.at>
+#
+#
+# License: BSD (3-clause)
+
+
+import inspect
+import os
+import os.path as op
+from numpy.testing import assert_allclose, assert_equal, assert_array_almost_equal
+import mne
+from mne.datasets import testing
+from mne.io.curry import read_raw_curry
+from mne.io.ctf import read_raw_ctf
+from mne.io.bti import read_raw_bti
+
+FILE = inspect.getfile(inspect.currentframe())
+
+curry_data_dir = op.join(op.dirname(op.abspath(FILE)), 'data')
```

Comment:
we don't use `inspect` we use `__file__` directly. Either way, we should move the files and use using `testing.data_path`